tgk/commands/issue-triage.ts

```
@@ -0,0 +1,196 @@
+  console.log('─'.repeat(50));
+}
+
+function displayDetailedAnalysis(analysis: TriageResult): void {
+  console.log('\n🔍 Detailed Analysis:');
+  console.log(JSON.stringify(analysis, null, 2));
+}
+
+// Types
+export interface GitHubIssue {
+  number: number;
+  title: string;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'GitHubIssue'. Consider to delete it or rename it.

'GitHubIssue' is defined here:


(lint/suspicious/noRedeclare)